rz-sbc/rzsbc_yocto.sh

```diff
@@ -38,6 +38,7 @@ function guideline() {
     echo "========="
     echo "Build yocto"
     echo " ./rzsbc_yocto.sh build <target_dir>"
+    echo " ./rzsbc_yocto.sh build-sdk <target_dir>"
```

Comment:
In the guideline function, there are two build options. The first is for common build with QT support and the second is for SDK build. So let's change to:

`./rzsbc_yocto.sh <target_build> <target_dir>`

then clarify the <target_build>.